src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double width;
+    private double height;
+
+    public Rectangle(String color, double width, double height) {
+        super(color);
+        this.width = width;
+        this.height = height;
+    }
+
+    public Rectangle(String color) {
+        super(color);
+    }
```

Comment:
The constructor 'Rectangle(String color)' seems unnecessary as it does not set the 'width' and 'height' properties. Consider removing it if it's not used or ensure it properly initializes the object.